pkg/pillar/cmd/ledmanager/ledmanager.go

```diff
@@ -761,7 +783,7 @@ func doLedAction(ledName string, turnon bool) {
 	var brightnessFilename string
 	var b []byte
 	if turnon == true {
-		b = []byte("1")
+		b = MaxLEDBrightness(ledName)
```

Comment:
Since you only read the first LED, one not do that on startup and save the value?
(The blinking pattern means running this code quite often.)

Reply:
I read the first LED name when the argument is a list, but eventually the function can be called different times with different LED names, so saved value will not be consistent. This functions reads from sysfs (memory implemented), so there is no substantial overhead...

Reply:
OK, but doesn't that mean that we should pass a single ledName to the Max function? Do the callers already ensure that? If so we don't need the check and comment (but might make sense to check for ',' and log.Fatal in that case since it would indicate an internal code bug.)

Reply:
OK, but doesn't that mean that we should pass a single ledName to the Max function? Do the callers already ensure that? If so we don't need the check and comment (but might make sense to check for ',' and log.Fatal in that case since it would indicate an internal code bug.)

Reply:
Yes @eriknordmark , you are right, doLedBlilnk() and doLedAction() actually already expect one LED, so we don't need the check actually... I will update the PR...